run/basics_test.go

```diff
@@ -84,3 +84,29 @@ func TestOctopus3branches(t *testing.T) {
 			"refs/heads/branch3\n"+
 			"\nCommit created by git-octopus "+VERSION+".")
 }
+
+func TestOneBranchFastForward(t *testing.T) {
+	//given
+	context, _ := CreateTestContext()
+	repo := context.Repo
+	defer test.Cleanup(repo)
+
+	repo.Git("checkout", "-b", "feature/test")
+	writeFile(repo, "testFeature", "")
+	repo.Git("add", "testFeature")
+	repo.Git("commit", "-m", "add testFeature")
+	repo.Git("rev-parse", "HEAD")
+	repo.Git("checkout", "master")
+
+	//when
+	Run(context, "feature/*")
+
+	//then
+	// feature/test should be merged in master
+	actual, _ := repo.Git("branch", "--contains", "feature/test")
+	assert.Contains(t, actual, "master")
```

Comment:
Given that the fast forward logic has been removed, this test should assert that the behavior is still working. Yeah.. it wasn't tested originally.
You could replace this assertion with something more precise:
```go
repo.Git("commit", "-m", "add testFeature")
testFeatureSha1, _ := repo.Git("rev-parse", "HEAD")

...
// Assert that master has been fast forwarded to feature/test
masterSha1, _ := repo.Git("rev-parse", "HEAD")
assert.Equals(t, testFeatureSha1, masterSha1)
```